app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,390 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+:root {
+  --primary-color: #351d77;
+  --secondar-color: #ec91c4;
+  --body: #e8f0e4;
+  --white: #fff;
+  --yellow: #e9b14c;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  font-size: 16px;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  background-color: var(--body);
+}
```

Comment:
- [ ] Please make sure to follow the design's typography, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.) requirement. You can use one of [these](https://www.learnui.design/blog/proxima-nova-similar-fonts.html) fonts which are free and similar to the font used in the design.
- [ ] As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=guidelines%2C%20including%3A-,Colors.,-Typography%3A%20font%20face) requirement, kindly follow the same design colors that were chosen for this design. You can find the exact color code written in the design.

Reply:
Noted with thanks!